packages/gatsby-source-wordpress/README.md

```diff
@@ -106,6 +121,33 @@ plugins: [
 ]
 ```
 
+## Fetching Data: WordPress REST API Route Selection
+
+The `gatsby-source-wordpress` plugin will fetch the most commonly used [WordPress data structures](https://developer.wordpress.org/rest-api/reference/#rest-api-developer-endpoint-reference):
+
+* Posts
+* Pages
+* Media
+* Categories
+* Tags
+* Users
+* Taxonomies
+
+By default, it will ignore all other endpoints listed in the root `/wp-json` endpoint.
+
+To customize the routes fetched, two configuration options are available: `includeRoutes` for whitelisting and `excludeRoutes` for blacklisting. Both options expect an array of glob patterns. Glob matching is done by [minimatch](https://github.com/isaacs/minimatch). To test your glob patterns, [use this tool](http://pthrasher.github.io/minimatch-test/).
```

Comment:
I don't remember right now - do we print out urls without `verbose` option?

Reply:
No, only the type and how many entities [were fetched](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-wordpress/src/fetch.js#L257).